light/client.go

```diff
@@ -861,7 +861,7 @@ func (c *Client) cleanupAfter(height int64) error {
 }
 
 func (c *Client) updateTrustedLightBlock(l *types.LightBlock) error {
-	c.logger.Debug("Updating trusted light block", "light block", l)
+	c.logger.Debug("updating trusted light block", "light block", l)
```

Comment:
this won't get logged because logger does not allow whitespaces in key names ("light block")